[2023-04-14 17:01:52,930] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.prepare_data manual__2023-04-14T17:01:51.120303+00:00 [queued]>
[2023-04-14 17:01:52,936] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.prepare_data manual__2023-04-14T17:01:51.120303+00:00 [queued]>
[2023-04-14 17:01:52,937] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:01:52,938] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 17:01:52,938] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:01:52,946] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): prepare_data> on 2023-04-14 17:01:51.120303+00:00
[2023-04-14 17:01:52,951] {standard_task_runner.py:52} INFO - Started process 229 to run task
[2023-04-14 17:01:52,954] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'prepare_data', 'manual__2023-04-14T17:01:51.120303+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmppar9g9d0', '--error-file', '/tmp/tmphweh7sij']
[2023-04-14 17:01:52,956] {standard_task_runner.py:80} INFO - Job 82: Subtask prepare_data
[2023-04-14 17:01:53,007] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.prepare_data manual__2023-04-14T17:01:51.120303+00:00 [running]> on host d014c8b814c7
[2023-04-14 17:01:53,062] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=prepare_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T17:01:51.120303+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T17:01:51.120303+00:00
[2023-04-14 17:01:53,063] {logging_mixin.py:115} INFO - Preparing Data
[2023-04-14 17:01:53,064] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 17:01:53,075] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=prepare_data, execution_date=20230414T170151, start_date=20230414T170152, end_date=20230414T170153
[2023-04-14 17:01:53,130] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 17:01:53,156] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
